<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>icFlow4: 2D collision simulation</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSave_Mesh"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionUse_Initial_State"/>
    <addaction name="actionCurrent_Space"/>
   </widget>
   <widget class="QMenu" name="menuTools_2">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="action_camera_reset"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionRemesh"/>
    <addaction name="actionSwap_Buffers"/>
    <addaction name="separator"/>
    <addaction name="actionClear_Velocity"/>
   </widget>
   <widget class="QMenu" name="menuSetup">
    <property name="title">
     <string>Setup</string>
    </property>
    <addaction name="actionIndentation"/>
    <addaction name="actionShear"/>
    <addaction name="actionStretch"/>
    <addaction name="actionSelf_collision"/>
    <addaction name="actionCZs"/>
    <addaction name="actionCZs_fracture"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuTools_2"/>
   <addaction name="menuSetup"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_GotoStep0"/>
   <addaction name="separator"/>
   <addaction name="action_simulation_start"/>
   <addaction name="action_simulation_single_step"/>
   <addaction name="separator"/>
  </widget>
  <action name="action_camera_reset">
   <property name="text">
    <string>Reset Camera</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_simulation_start">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/ic/icons/media-playback-start.svg</normaloff>
     <normalon>:/ic/icons/media-playback-pause.svg</normalon>:/ic/icons/media-playback-start.svg</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/but-quit.png</normaloff>:/icons/but-quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_simulation_single_step">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/ic/icons/system-log-out.svg</normaloff>:/ic/icons/system-log-out.svg</iconset>
   </property>
   <property name="text">
    <string>Single Step</string>
   </property>
   <property name="toolTip">
    <string>Single step</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_GotoStep0">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/ic/icons/go-first.svg</normaloff>:/ic/icons/go-first.svg</iconset>
   </property>
   <property name="text">
    <string>GotoStep0</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionSave_Mesh">
   <property name="text">
    <string>Save Mesh...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionUse_Initial_State">
   <property name="text">
    <string>Initial Space</string>
   </property>
  </action>
  <action name="actionRemesh">
   <property name="text">
    <string>Remesh</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionSwap_Buffers">
   <property name="text">
    <string>Swap Buffers</string>
   </property>
  </action>
  <action name="actionClear_Velocity">
   <property name="text">
    <string>Clear Velocity</string>
   </property>
  </action>
  <action name="actionCurrent_Space">
   <property name="text">
    <string>Current Space</string>
   </property>
  </action>
  <action name="actionIndentation">
   <property name="text">
    <string>Indentation</string>
   </property>
  </action>
  <action name="actionShear">
   <property name="text">
    <string>Shear</string>
   </property>
  </action>
  <action name="actionStretch">
   <property name="text">
    <string>Stretch</string>
   </property>
  </action>
  <action name="actionSelf_collision">
   <property name="text">
    <string>Self-collision</string>
   </property>
  </action>
  <action name="actionCZs">
   <property name="text">
    <string>CZs</string>
   </property>
  </action>
  <action name="actionCZs_fracture">
   <property name="text">
    <string>CZs-fracture</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
